announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm

```diff
@@ -390,6 +410,9 @@
 								#else
 									<img src = "#imageLink("sakai/attachments.gif")" border="0" alt="$tlang.getString('gen.attach')" />
 								#end
+								#if ($highlight == "true")
```

Comment:
I don't like this, maybe we can do it via CSS.

Reply:
Ok. I will do it via css in this and the others